src/home/homed-home.h

```diff
@@ -209,7 +214,13 @@ int home_killall(Home *h);
 
 int home_augment_status(Home *h, UserRecordLoadFlags flags, UserRecord **ret);
 
-int home_create_fifo(Home *h, bool please_suspend);
+typedef enum {
+        HOME_FIFO_PLEASE_SUSPEND,
+        HOME_FIFO_DONT_SUSPEND,
+        HOME_FIFO_INHIBIT_SUSPEND,
```

Comment:
we generally want that enums like this end in

_HOME_FIFO_TYPE_MAX,
_HOME_FIFO_TYPE_INVALID = -EINVAL,